daal4py/sklearn/ensemble/_forest_0_23.py

```diff
@@ -0,0 +1,629 @@
+        #if self.criterion != "gini":
+        #    warnings.warn(_class_name + ' currently only supports criterion="gini"')
+        if isinstance(self.min_samples_leaf, numbers.Integral):
+            if not 1 <= self.min_samples_leaf:
+                raise ValueError("min_samples_leaf must be at least 1 "
+                                 "or in (0, 0.5], got %s"
+                                 % self.min_samples_leaf)
+        else:  # float
+            if not 0. < self.min_samples_leaf <= 0.5:
+                raise ValueError("min_samples_leaf must be at least 1 "
+                                 "or in (0, 0.5], got %s"
+                                 % self.min_samples_leaf)
+        if isinstance(self.min_samples_split, numbers.Integral):
+            if not 2 <= self.min_samples_split:
+                raise ValueError("min_samples_split must be an integer "
+                                 "greater than 1 or a float in (0.0, 1.0]; "
+                                 "got the integer %s"
+                                 % self.min_samples_split)
+        else:  # float
+            if not 0. < self.min_samples_split <= 1.:
+                raise ValueError("min_samples_split must be an integer "
+                                 "greater than 1 or a float in (0.0, 1.0]; "
+                                 "got the float %s"
+                                 % self.min_samples_split)
+        if not 0 <= self.min_weight_fraction_leaf <= 0.5:
+            raise ValueError("min_weight_fraction_leaf must in [0, 0.5]")
+        if self.min_impurity_split is not None:
+            warnings.warn("The min_impurity_split parameter is deprecated. "
+                          "Its default value has changed from 1e-7 to 0 in "
+                          "version 0.23, and it will be removed in 0.25. "
+                          "Use the min_impurity_decrease parameter instead.",
+                          FutureWarning)
+
+            if self.min_impurity_split < 0.:
+                raise ValueError("min_impurity_split must be greater than "
+                                 "or equal to 0")
+        if self.min_impurity_decrease < 0.:
+            raise ValueError("min_impurity_decrease must be greater than "
+                             "or equal to 0")
+        if self.max_leaf_nodes is not None:
+            if not isinstance(self.max_leaf_nodes, numbers.Integral):
+                raise ValueError("max_leaf_nodes must be integral number but was "
+                                 "%r" % self.max_leaf_nodes)
+            if self.max_leaf_nodes < 2:
+                raise ValueError(("max_leaf_nodes {0} must be either None "
+                                  "or larger than 1").format(self.max_leaf_nodes))
+
+def _fit_regressor(self, X, y, sample_weight=None):
+    print("DAAL REGRESSOR FIT")
+    return super(RandomForestRegressor, self).fit(X, y, sample_weight=sample_weight)
+
+def _predict_regressor(self, X):

```

Comment:
All functions may be left as is or added in class, for my point of view both variant are possible